README.md

```diff
@@ -112,21 +112,41 @@ Email: john@barong.io, password: Am8icnzEI3d!
 
 All the OpenDAX deployment files have their confguration stored in `config/app.yml`.
 
-Feel free to fill it out with correct values:
-
-| Parameter         | Description                                      |
-| ----------------- | ------------------------------------------------ |
-| `app.name`        | Global application name                          |
-| `app.domain`      | Base domain name to be used                      |
-| `ssl.enabled`     | Enable SSL certificate generation                |
-| `ssl.email`       | Email address to use for SSL generation requests |
-| `images`          | Application images tags                          |
-| `vendor`          | Frontend application Git repo URL                |
-| `render_protect`  | Enable read-only mode for rendered files         |
-
-    Twilio SMS service is disabled by default, feel free to enable it in the `twilio` section of the config
-
-    Note: You can protect all the rendered files from being edited so that you wouldn't lose your changes upon re-rendering templates in the future by setting `render_protect` to `true`
+#### app.yml
+
+The following table lists the configurable parameters of the config/app.yml configuration file and its default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`app.name` | global application name | `"OpenDax"`
+`app.domain` | base domain name to be used | `app.local`
+`subdomain` | subdomain of your application | `www`
+`render_protect` | enable read-only mode for rendered files | `false`
+`csrfEnabled` | enable csrf token on barong | `false`
+`ssl.enabled` | enable SSL certificate generation | `false`
+`ssl.email` | email address to use for SSL generation requests | `"support@example.com"`
+`images` | application images tags
+`vendor.frontend` | vendor is a tool to develop front with opendax, put there a link to you front repo | `git@github.com:openware/baseapp.git`
+`vault.token` | root token to unseal the vault | `changeme `
```

Comment:
We don't use root token, to unseal vault.
Just put "Vault root token"

Reply:
Let's put Vault auth token, it doesn't have to be root :)